pkg/config/signer_test.go

```diff
@@ -0,0 +1,221 @@
+/*
+Copyright IBM Corp. All Rights Reserved.
+
+SPDX-License-Identifier: Apache-2.0
+*/
+
+package config
+
+import (
+	"crypto/ecdsa"
+	"crypto/rand"
+	"crypto/x509"
+	"io"
+	"math/big"
+	"testing"
+
+	"github.com/gogo/protobuf/proto"
+	"github.com/hyperledger/fabric-protos-go/msp"
+	. "github.com/onsi/gomega"
+)
+
+var (
+	publicKey = `
```

Comment:
how have these fixtures been generated? Ideally we should just run go generate in the folder to re-generate them. Failing that, can we at least have a comment describing how to re-generate them?

Reply:
Good catch. Didn't mean to include these fixtures instead of generating them. We'll fix this.

Reply:
Looking into this now. I'll update the PR when things are ready.

Reply:
Done. Adapted from the code here: https://golang.org/src/crypto/tls/generate_cert.go